src/components/ManageVaults.tsx

```diff
@@ -0,0 +1,23 @@
+import { useVaultStore } from 'store/vaults';
+import VaultSummary from 'components/VaultSummary';
+import { useAtomValue } from 'jotai';
+import { displayFunctionsAtom } from 'store/app';
+
+const ManageVaults = () => {
+  const vaults = useVaultStore(state => state.vaults);
+  const displayFunctions = useAtomValue(displayFunctionsAtom);
```

Comment:
a little confusing to me that displayFunctions is the condition for rendering. What exactly is "loading" when they're not available?

Reply:
`displayFunctions` is derived from `brandToInfo` which is fetched from `agoricNames.brands`. I guess it's not super clear just from it being nullable that it's something that we want to wait to load. I'm considering switching the types to something more like [react-query ](https://react-query-v3.tanstack.com/reference/useQuery#_top) where you have not just `data?` but `{ data, isLoading, isError}` etc.

Reply:
> something more like [react-query ](https://react-query-v3.tanstack.com/reference/useQuery#_top)where you have not just data? but { data, isLoading, isError} etc.

SGTM